src/modules/SX126x/SX126x.cpp

```diff
@@ -1169,6 +1187,16 @@ uint32_t SX126x::getTimeOnAir(size_t len) {
   }
 }
 
+void SX126x::implicitHeader(size_t len) {
```

Comment:
Please add both of these to keywords.txt using true tab.

Reply:
Also, both of these methods should check active modem - implicit/explicit header only applies to LoRa.

Reply:
I believe the active modem should be checked at RX/TX -- it could change after calling these methods. Though, if we're OK with that risk it might be simpler.

Reply:
Modem is already checked at Rx/Tx, this is purely for safety reasons - if the user attempts to change LoRa-only setting while FSK modem is active, it's a sign they're doing something wrong. I also still think both of these should call setPacketParams to send the changed header type into the radio itself - that's probably the safest option.